### PR TITLE
Initialization of the database after creation and implementation of "about" and "status" commands.

### LINKED ISSUE #6: Implement "create" command
Write the python script that initializes the database using SQLAlchemy.

### LINKED ISSUE #9: Implement "about" command
This command will print a brief description of the project (license, contributors etc.).

### LINKED ISSUE #17: Initialize DB_INFORMATION table when the database is created

### LINKED ISSUE #19: Initialize SCAN_CODE table when the database is created
This table will be initialized when the database is created and it will not be changed after the creation.
It will contain descriptions/error messages regarding the result of a scan.

### LINKED ISSUE #20: Initialize HASH_FUNCTION table when the database is created
This table will contain information about the hash functions supported by hashesDB.
The first version will support the functions provided by the built-in  _hashlib_ module.
It will also support ssdeep fuzzy hash function.

### LINKED ISSUE #22: Implement "status" command
This command will print the database currently used. Moreover, it will inform the user if there are unsaved changes. Finally, it will display the working directory from which the relative paths begin.

### PR DESCRIPTION
**Pull request by gkosm314**
This pull request contains commits that resolve issue #19, resolve issue #20.
Since issue #17 has been resolved by a previous pull request, this pull request subsequently resolves issue #6.

Since we haven't discussed the code that closes issues #17 and #6, I will add comments to the discussion section of the aforementioned issues, explaining how the code I have written so far works.

This pull request contains functions that populate the database immediately after its creation. In more detail, it contains a function that fills the HASH_FUNCTION table with records that correspond to the default hash functions contained in the built-in hashlib module. Additionally, xxh3_64 and xxh3_128 hash functions, as well as the ssdeep fuzzy hash function were added. The Software Heritage Identifier(SWHID) is not added to the table, since SWHIDs will be stored at the SWH_INFO table (maybe we should discuss this part, if you disagree).

Finally, a function that implements the "about" command was added. This function prints a paragraph that briefly describes the project.

### CODE REVIEW

**Comment by gkosm314** [comment]
I forgot to mention that more scan codes will be added to the SCAN_CODE table, once the "scan" command is implemented an more specific errors emerge.

**Comment by gkosm314** [comment]
I also made some minor enhancements that make it possible to close issue #22.

File: src/app.py [hunk 1]
```diff
@@ -8,6 +9,9 @@ def __init__(self, used_database_path_param = None):
 		#By default our app runs sequentially
 		self.max_threads = 1
 
+		#Set the working directory
+		self.working_directory = getcwd()
```
Comment (zvr) [inline src/app.py:13]: not sure why this is useful, but...

Comment (gkosm314) [reply src/app.py:13]: I think it would be useful for the user to know which the working directory is, so that he can use paths relative to that path.

Comment (zvr) [reply src/app.py:13]: I don't think it will be ever used, but we can always delete it aferwards

**Pull request merged**